style.css

```diff
@@ -103,264 +83,172 @@
   }
 
-  .contactLink .resum {
-    width: 138px;
-    height: 24px;
-    font-style: normal;
-    font-weight: 500;
-    font-size: 17px;
-    line-height: 24px;
+  .framWork ul li {
+    color: rgb(34, 30, 30);
     display: flex;
-    align-items: center;
-    text-align: center;
-    letter-spacing: 0.03em;
-    color: #396df2;
-    border: 1px solid #396df2;
+    gap: 10px;
+    background: #dfe1e6;
+    margin: 10px;
     position: relative;
-    left: 20px;
-    margin-top: 10px;
-    margin-bottom: 10px;
-    padding: 7px 20px;
-    border-radius: 8px;
-  }
-
-  .contactLink .resum:hover {
-    background: #396df2;
-    color: white;
+    right: 10%;
   }
 
-  .framWork ul li {
+  .techLanguage ul li {
     display: flex;
-    flex-direction: row;
-    color: brown;
-    gap: 15px;
-    margin-bottom: 10px;
-    margin-left: 20px;
-    width: 303px;
-    height: 66.72px;
-    padding: 8px 20px;
-    background: #f7f7f9;
-    border-radius: 8px;
+    justify-content: space-between;
+    border-bottom: 1px solid #dfe1e6;
   }
 
-  .framWork p {
-    color: #172b4d;
+  .techLanguage ul li img {
+    width: 30px;
+    height: 30px;
     position: relative;
-    top: 20px;
+    top: 10px;
   }
 
-  .framWork .code {
-    display: flex;
-    gap: 180px;
-    margin-top: 20px;
-    margin-bottom: 50px;
-    position: relative;
-    left: 40px;
+  .Contacting {
+    font-family: "Poppins", sans-serif;
+    background-color: #6070ff;
+    text-align: center;
+    height: 846px;
+    padding: 24px;
+    border-top-left-radius: 80px;
+    background-image: url("img/contact-mobile.svg");
+    background-position: right 50px;
+    background-repeat: no-repeat;
   }
 
-  .framWork .code img {
-    width: 5%;
-    height: 5%;
-    position: relative;
-    top: 10px;
+  .Contacting h1 {
+    color: #fff;
+    font-size: 40px;
+    font-weight: 700;
+    margin-top: 103px;
   }
 
-  .techLanguage ul li {
-    display: flex;
-    gap: 180px;
-    margin-top: 20px;
-    position: relative;
-    left: 40px;
-    margin-bottom: 40px;
-    border-bottom: 2px solid #dfe1e6;
-    padding-bottom: 20px;
-    width: 300px;
+  .Contacting p {
+    font-size: 20px;
+    font-weight: 400;
+    color: #ebebff;
   }
 
-  .techLanguage #oneImg {
-    width: 30px;
-    height: 30px;
+  .formController {
+    margin: 8px auto;
   }
 
-  .techLanguage .move {
-    position: relative;
-    left: 50px;
+  .formController input {
+    width: 90%;
+    height: 48px;
+    border-radius: 8px;
+    border: none;
+    outline: none;
+    padding: 4px 8px;
   }
 
-  .techLanguage .oneMove {
-    position: relative;
-    left: -12px;
+  .formController textarea {
+    width: 90%;
+    height: 170px;
+    border-radius: 8px;
+    border: none;
+    outline: none;
+    padding: 4px 8px;
+  }
+
+  .Contacting .btn {
+    display: block;
+    height: 48px;
+    width: 142px;
+    border-radius: 8px;
+    color: #6070ff;
+    font-size: 17px;
+    border: 1px solid #6070ff;
   }
```

Comment:
- Hey, let's kindly align the button correctly to be on the same line as the inputs and textarea of the form to improve the design to be close to the Figma design as much as we can:

*your design*
![image](https://user-images.githubusercontent.com/59397159/235718313-b1befd7d-b2fa-4e72-ad76-f14c25043deb.png)

*Figma design*
![image](https://user-images.githubusercontent.com/59397159/235718611-21654b8a-49a4-4d1b-9b00-9b41c0c2349c.png)

Reply:
- Hey, let's add the missing interaction states of each button in your webpage to indicate the (hover, disabled, enabled, active) to improve the user experience when the buttons are clicked:

![image](https://user-images.githubusercontent.com/59397159/235633008-a3613ba4-b895-4b39-89a7-af9eab9c12f5.png)